h2/src/main/org/h2/command/ddl/AlterTableAlterColumn.java

```diff
@@ -557,8 +568,13 @@ public void setIfNotExists(boolean ifNotExists) {
         this.ifNotExists = ifNotExists;
     }
 
-    public void setNewColumns(ArrayList<Column> columnsToAdd) {
-        this.columnsToAdd = columnsToAdd;
+    @Override
+    public void addColumn(Column column) {
+        ArrayList<Column> columnsToAdd = this.columnsToAdd;
```

Comment:
no need for a local variable here, just reference the instance field

Reply:
OK, this optimization is not useful here. I also remove such local variables in two more similar places.